lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -23,6 +24,8 @@ const rearrangeCoefficient = require('./rearrangeCoefficient');
 const SIMPLIFICATION_FUNCTIONS = [
   // multiplication by 0 yields 0
   reduceMultiplicationByZero,
+  // division by zero should fail
```

Comment:
I think error checks should actually go in a different tree search. Can you make a folder `errorsSearch` and put `handleDivisionByZero` in that tree search instead? And that search would happen first, before anything else